lib/internal/process/per_thread.js

```diff
@@ -296,7 +295,7 @@ function buildAllowedFlags() {
   }
 
   function isAccepted(to) {
-    if (!StringPrototypeStartsWith(to, '-') || to === '--') return true;
+    if (!to || to[0] !== '-' || to === '--') return true;
```

Comment:
!to.length is better in here I suppose.

Reply:
Also why not `to?.[0] !== '-'`

Reply:
I use `!expression` to cover the empty string as well, lately I've been unable to detect a performance difference between `!string` and `!string.length`

[ref](https://perf.link/#eyJpZCI6Inl1a2tuc2N0dzc1IiwidGl0bGUiOiJGaW5kaW5nIG51bWJlcnMgaW4gYW4gYXJyYXkgb2YgMTAwMCIsImJlZm9yZSI6ImNvbnN0IGRhdGEgPSBbLi4uQXJyYXkoMTAwMCkua2V5cygpXS50b1N0cmluZygpXG5sZXQgYSA9IDBcbmxldCBiID0gMFxubGV0IGMgPSAwIiwidGVzdHMiOlt7Im5hbWUiOiJUZXN0IENhc2UiLCJjb2RlIjoiaWYgKGRhdGEubGVuZ3RoICE9PSAwKSArK2MiLCJydW5zIjpbMTQ4NTcwMDAsNDg3NDAwMCwyNzgyMDAwLDQ2ODUwMDAsMTkwNzEwMDAsMjAxNjIwMDAsMTcwMjYwMDAsMjczMDAwMCwxNzE4MDAwMCwxMTQ5MDAwLDExNDkwMDAsNzUyMTAwMCwxODk1MDAwMCwxOTY5NDAwMCwxNTE2NTAwMCwxMTE4MDAwMCwyODMzMDAwLDc5NjQwMDAsMTYxODUwMDAsMTE0NDUwMDAsNDcxNTAwMCw2MTE2MDAwLDUxMDAwLDEyMDQ0MDAwLDkwNTEwMDAsMTc4ODAwMCwxMTA3MTAwMCwzMjE1MDAwLDE5NjM4MDAwLDEyNTM0MDAwLDE0MDYxMDAwLDcwMDAwMCwyODMzMDAwLDIzNTAwMDAsMTA1NjYwMDAsNjY5NDAwMCw5NzQ3MDAwLDU3OTUwMDAsOTgwNTAwMCw5NzUwMDAwLDEwMzgzMDAwLDI4ODIwMDAsMTE2NjkwMDAsMzkzMDAwLDY2OTEwMDAsMTU1OTgwMDAsMzYwMDAsNDAzNDAwMCwxNTQ2MDAwLDI5NzUwMDAsNDM2MzAwMCwyMTUxMjAwMCwxODA4MDAwMCwzNjEwMDAwLDg5NzIwMDAsMTczMjAwMCwxMjM4MjAwMCwxMTQ5MDAwLDE1NzA2MDAwLDE2NzAzMDAwLDExNDkwMDAsMTUzMTAwMCwxMTQ5MDAwLDEyNTkwMDAsMjAyNjMwMDAsODQ3MjAwMCwxMTQ5MDAwLDExMzA3MDAwLDE4NTI4MDAwLDU5MjcwMDAsMjExNzEwMDAsNTY1NzAwMCw1MjYwMDAwLDEyNzI1MDAwLDExNDkwMDAsMTQxMzAwMCwyNjk3MDAwLDEzOTM1MDAwLDE0NDMwMDAsMTE0OTAwMCwzNTE2MDAwLDc2MDcwMDAsNDc0MzAwMCwxOTUwNTAwMCwxODU2ODAwMCw3NTEzMDAwLDczNTkwMDAsMzk1MzAwMCwxMDYxNjAwMCw3ODkyMDAwLDExNDkwMDAsMTYwMjUwMDAsNDEwMDAsMjczNTAwMCwzNjEwMDAwLDI4MjAwMCw4MjEwMDAsMTA3NDYwMDAsMTE2NDMwMDAsMTIyNjYwMDBdLCJvcHMiOjgxNzQ2NTB9LHsibmFtZSI6IkZpbmQgaXRlbSAxMDAiLCJjb2RlIjoiaWYgKGRhdGEubGVuZ3RoKSArK2IiLCJydW5zIjpbMTA3MjgwMDAsMTYyODAwMCwxMzg5NDAwMCwxMzEyMDAwLDE0NDU5MDAwLDE0MjgwMDAwLDExMjg3MDAwLDI4NDAwMDAsMTIxODIwMDAsMTg1ODEwMDAsMTE5NzAwMCwzNTI1MDAwLDEzNTgzMDAwLDE1NjQwMDAwLDYxMTMwMDAsNzM4NjAwMCw5MzUwMDAsMzA4OTAwMCwxMDc0MzAwMCw2NDY0MDAwLDE4MzEwMDAsMzE3MDAwMCwxNDI5MDAwLDc5MjMwMDAsMTE5NzAwMCwxNjc2OTAwMCw4OTc3MDAwLDMzMjYwMDAsOTgzMzAwMCw3MTUxMDAwLDg0NTkwMDAsMTUyMzcwMDAsMTg2NzAwMDAsMjgzMDAwLDU1NjEwMDAsNDQ0NjAwMCw1MTM2MDAwLDE0ODkwMDAsMzUxNzAwMCwzMjcxMDAwLDczNzMwMDAsMTE5NzAwMCw1MDYzMDAwLDEzNTI4MDAwLDI1MDYwMDAsMTAyMTMwMDAsMTUwMjIwMDAsMjA1NzgwMDAsMjA1MTIwMDAsNDIwMDAsNzc0MDAwLDQwMjAwMDAsMTIxMzYwMDAsMTAyMDAwMCwyMzM1MDAwLDE3NTA2MDAwLDQxNTQwMDAsMTEzMDAwLDEyMzk0MDAwLDk3MzAwMCwxNDYzMDAwMCwyMDA2NDAwMCwxNDQ1MDAwMCwxNzgxMTAwMCwxNDUxODAwMCwzNTE0MDAwLDYwNDAwMCwxMTAyODAwMCwxMDAwNzAwMCwxOTM1MDAwMCwxNTAzNDAwMCwyNTk0MDAwLDEzODA5MDAwLDg0MzYwMDAsMTUyMzgwMDAsODQwMDAwLDI3MDgwMDAsMTI2NDcwMDAsMTgxMDQwMDAsMTE5NzAwMCwxMTEwMDAsMjc1ODAwMCwxMTk3MDAwLDIwMjc0MDAwLDE4Njg0MDAwLDY5NDkwMDAsMTA4MDUwMDAsMzE4MzAwMCwzOTU4MDAwLDM1NjUwMDAsODgzMDAwMCwxMDY4MTAwMCwxMTA4NDAwMCwxOTQwMDAsMTE5NzAwMCwxMjcxOTAwMCwxNjgwMzAwMCw0ODcyMDAwLDI3NzUwMDAsMTIxNjEwMDBdLCJvcHMiOjgxODM4MzB9LHsibmFtZSI6IkZpbmQgaXRlbSA4MDAiLCJjb2RlIjoiaWYgKGRhdGEpICsrYSIsInJ1bnMiOls1NTU2MDAwLDE4NjM1MDAwLDk1OTEwMDAsMTg0MDUwMDAsMzI5NzAwMCw5NjczMDAwLDUxODMwMDAsMzAwMCw2OTY1MDAwLDEzNjUyMDAwLDE3NTIyMDAwLDcxMzAwMCw2MTE5MDAwLDg3MjQwMDAsNDY5NDAwMCwyODA3MDAwLDEzMDg0MDAwLDMzODAwMCw1MjE1MDAwLDIwMjUyMDAwLDE5MjgxMDAwLDE3MTUyMDAwLDEyNTAyMDAwLDE1NDgwMDAsMTI1NzAwMCw4OTkwMDAwLDc3ODgwMDAsMTE5MDAwMCwyOTk5MDAwLDI2NDIwMDAsMjg5MjAwMCwxMTExNzAwMCwxNDU0NjAwMCwxMTY2MDAwMCwxNDg4MDAwLDE4MzcwMDAsMTI1NzAwMCw1NTYwMDAsODA1MDAwLDE5NTgwMDAwLDIxMTYwMDAsNzQ1MjAwMCwxMjU3MDAwLDkyNTYwMDAsMTU2OTgwMDAsMzk2ODAwMCwxMDA3MjAwMCwxNTEwMjAwMCwxNjA2ODAwMCwxNDU5MTAwMCwxNzU3NzAwMCwxMTIwMDAwLDY0NzkwMDAsMTUwNjUwMDAsMTcwMDAsMTMxOTgwMDAsNDE0ODAwMCwxMjMwNTAwMCw1NTc0MDAwLDExOTExMDAwLDEyMTk3MDAwLDEyOTczMDAwLDc1MDgwMDAsMTE5OTYwMDAsODg2NjAwMCw1NjcwMDAsMTA2OTcwMDAsNjM5NjAwMCw0NTQzMDAwLDEzMDg0MDAwLDkyMjgwMDAsMTMxNjgwMDAsMTMzMjIwMDAsNTU3NzAwMCwxMDE3OTAwMCwxMjk1NjAwMCwyODY3MDAwLDEwNTg3MDAwLDExODgwMDAwLDE3NzAwMCwxOTA1MzAwMCw1NzI2MDAwLDE2Mjc2MDAwLDIwMjU2MDAwLDc1MjQwMDAsNTkxODAwMCw2ODgzMDAwLDM0MjgwMDAsMTI1NzAwMCw2MTk1MDAwLDE4MTYwMDAsMTI1NzAwMCwxMzkzMTAwMCw2MTQ5MDAwLDEyMDIwMDAwLDgxNjgwMDAsOTMzMTAwMCwyMDYzNjAwMCwyNzQ4MDAwLDk4NzYwMDBdLCJvcHMiOjg1MTcwNTB9XSwidXBkYXRlZCI6IjIwMjQtMTAtMTZUMTc6MzI6MDEuNzQ2WiJ9)

But the intent is clearer, I'll change them to str.length conditions

Reply:
If the string is empty, we will hit undefined and a property chain lookup